src/components/select-field/props.js

```diff
@@ -4,5 +4,8 @@ export type SelectOptionValueProp = string | number | null;
 
 export type SelectOptionProp = {
     displayText: string,
+    hasWarning?: boolean,
```

Comment:
Let's make the same change you did for the avatar URL for these other new options here. `id` is OK b/c a `<select>` and all elements can have an ID. We want to separate out `hasWarning`, `isExternalUser`. Where did `avatarUrl` move to?